sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md

```diff
@@ -1,19 +1,14 @@
 # Release History
 
-## 1.0.0b4 (Unreleased)
+## 1.0.0b4 (2021-11-10)
 
 ### Features Added
 
 - Async version of `AvroSerializer` has been added under `azure.schemaregistry.serializer.avroserializer.aio`.
```

Comment:
@yunhaoling do you think any of these should be in other sections (not Features Added)?

Reply:
that's a good question, I will list the following under breaking changes as it would break exception handling.
···
  - `SchemaParseError` and `SchemaSerializationError` may be raised for errors when calling `serialize` on `AvroSerializer`.
  - `SchemaParseError` and `SchemaDeserializationError` may be raised for errors when calling `deserialize` on `AvroSerializer`.
···